ext/standard/string.c

```diff
@@ -5776,10 +5776,10 @@ static zend_string *php_str_rot13(zend_string *str)
 		const __m128i m_ = _mm_set1_epi8('m' + 1);
 		const __m128i _n = _mm_set1_epi8('n' - 1);
 		const __m128i z_ = _mm_set1_epi8('z' + 1);
-		const __m128i _A = _mm_set1_epi8('A' - 1);
-		const __m128i M_ = _mm_set1_epi8('M' + 1);
-		const __m128i _N = _mm_set1_epi8('N' - 1);
-		const __m128i Z_ = _mm_set1_epi8('Z' + 1);
+		const __m128i A_ = _mm_set1_epi8('A' - 1);
+		const __m128i _M = _mm_set1_epi8('M' + 1);
+		const __m128i N_ = _mm_set1_epi8('N' - 1);
+		const __m128i _Z = _mm_set1_epi8('Z' + 1);
```

Comment:
The previous naming here indicated whether the value was one below or one above the character. Now half the variable names (the lower-case ones) use a different convention from the others (the upper-case ones).

I'd suggest to change all 8 variables to something more explicit, like `a_minus_1`, `m_plus_1`, `n_minus_1`, `z_plus_1`, `A_minus_1` etc.

Reply:
Ok why not I did not dare to make too much changes ... will update soonish.